### ISSUE TITLE
Newbie Alert! Dipping my toe in...

### ISSUE THREAD

**Issue by damienhurrell** [kdechant/eamon#22]
I'm a very long-time user of Eamon (used to play on an Apple ][ clone in the 80s) and I've been enjoying rediscovering my old friends (and unfriendlies!) through Eamon Deluxe for a few years.  While playing Deluxe, I've been thinking how great it would be to be able to play Eamon on a web-based environment, with all the advantages of portability and accessibility that implies. And then I found Eamon remastered: Joy! Rapture! Mapmaking! Battles! Looting! I'm in heaven and so pleased that you've taken this project on.

I'd love to help, but unfortunately I haven't kept up with programming since I learned ADA for a computer science subject in the early 90s, for a degree that I then dropped out of. But I am kinda interested in learning to code again, so never say never.

But at the moment, I think my most significant contribution would be as a bug catcher and beta tester.  Therefore, I offer up the following observations:

1) I had a significant issue when trying to play away from home.  I could log in, start an adventure and move around, but as soon as I tried to attack an unfriendly (or manipulate an artifact) nothing happened. The system would not accept the command at all.   This was on IE 11, but everything worked perfectly on Chrome. Happy to provide more environment details if that helps.

2) Running my beginning character through the training ground, I missed an old foe: the Jade Warrior.  In the ED version, the Jade Warrior appears when the player opens the obsidian scroll case in the secret scroll room.  But in ER, while you can take the obsidian scroll case twice, the Warrior is conspicuous by it's absence.

3) secret doors are often not so secret, because they show as a possible direction to exit in the room description panel even before the player has noticed the hint and examined the suspiciously door shaped rock in the east wall (or whatever).  I would think that the secret direction should remain hidden in the room panel until the secret is out. 

4) including the room number in the description is going to make finding one's way through a labyrinth much, much easier. Labyrinths usually include unhelpful, vague or misleading descriptions that the player has to somehow work through.  Room numbering provides precise information that removes the element of uncertainty that a number of authors have used to good effect.  This is more a question of style, but it does make a difference to gameplay.

I hope these observations are helpful. Thankyou for Eamon Remastered: I hope that it continues on from strength to strength.

**Comment by kdechant** [comment]
On closer inspection, I noticed that my deployment system was failing, and that some of the most recent code wasn't making it onto the server. Try the Training Ground again. This time, the room numbers should be gone, and the Emerald Warrior should appear when you pick up the obsidian scroll case.

I also added a fix for the issue with partial command matching in IE. FYI, there are other issues in IE, like the lack of smooth scrolling in the output window, which are not easily fixable. I'm not planning to fix those, due to the fact that there are only a few IE users. I'll keep fixing critical game play issues in IE for a while, at least.

**Comment by damienhurrell** [comment]
I've had a run around the training ground, and yes, the room numbers are gone and the Emerald Warrior appears. Thankyou. Room numbers are no longer visible anywhere I've visited today

I'll have a play on IE11 when the opportunity presents itself and report back.

On the question of secret exits, I've had a run around a number of adventures and as far as I could tell, every secret door in every adventure was shown as a "visible exit" before I searched for and found said doors in gameplay.

This includes:
- the secret door leading to the Temple in the Beginner's Cave 
- the door shaped chunk of quartz, the crossing of the chasm, and the passage behind the west panel in the Cave of the Mind
- the secret path north from the oak tree in the beginner's forest
- the loose brick in the cell, the "funny rock" in the tee intersection, the entrance behind the poster to the treasury, and the door to the exit in the escape from orc lair
- the path through the cracks towards the flamethrower room and the passage behind the ice carving in the Orb of Polaris
- and so on
